src/include/daos_srv/vos_types.h

```diff
@@ -356,6 +356,8 @@ typedef struct {
 		struct {
 			/** record size */
 			daos_size_t		ie_rsize;
+			/** record size for the whole global single record */
```

Comment:
So io_rsize is set to the size of the extent store at this VOS?

Reply:
yes